client/app/bundles/course/admin/pages/LessonPlanItemSettings/index.jsx

```diff
@@ -76,48 +76,119 @@ class LessonPlanItemSettings extends React.Component {
     );
   }
 
-  renderLessonPlanItemSettingsTable() {
+  renderComponentSettingRow(setting) {
```

Comment:
As discussed, once we have settings for lesson plan events, we will have 3 tables. It might be neater to consolidate them into one table. Instead of a row for tab_name/event_type, we can concatenate names of sub-components to use as titles, similar to the entries currently in the lesson plan filter dropdown.

Reply:
Yes, will change this when implementing #2762.